### ISSUE TITLE
Linux load average also counts I/O

### ISSUE THREAD

**Issue by jdemeyer** [hpcugent/vsc_user_docs#120]
In section 8.4, the discussion about load average indicates that it's only about CPU usage, but this is not correct. In particular this is really wrong:

> Load averages do not include any processes or threads waiting on I/O,
networking, databases or anything else not demanding the CPU. It narrowly
focuses on what is actively demanding CPU time.

Threads waiting for disk I/O also contribute load average (networking does not). You can see this for example by running some I/O-intensive process like `tar`. This will cause the load average to increase.

A much better way to think of load average is as the number of processes which are not idle. They might be "not idle" because they are running or because they are waiting for I/O.

**Comment by stdweird** [comment]
@jdemeyer yes, this is a mistake in the text. load takes into account interruptable sleep, so processes "waiting for IO" (e.g. local filesystem or NFS) will make the load go up without increasing CPU load (well, iowait might also go up). i'm not sure what IO on GPFS does wrt load.
load measures processes/threads using or waiting in queue for cpu slot. ideally this is 1 process/thread using a complete core, and no other processes waiting for cpu.

but load is a complex metric and should not be used on it's own to judge ideal usage of resources. i think the text does a good job at making that clear in the beginning.

feel free to open a PR for a better phrasing :smile:
